marlowe-playground-client/src/Contrib/Halogen/State/Record.purs

```diff
@@ -0,0 +1,74 @@
+module Contrib.Halogen.State.Record where
```

Comment:
As mentioned before, I think that it would be good in the future to generalize this to a set of tools for creating field lenses and prisms from records and variants. Also, I think it doesn't make a lot of sense for this to be namespaced with `Halogen`, as you say below, it isn't really related to halogen... I'd call it a "peer-dependency" :)

Reply:
Yeah, I've got some working prototype here:

https://try.purescript.org/?gist=b7b3e768886639004c355250b7a6ea07

I want to provide a nicer way to specify signatures because signatures generated by compiler are awful (the `Lens` related alises are expanded etc.).